src/gluonts/model/forecast.py

```diff
@@ -318,7 +343,7 @@ def as_json_dict(self, config: "Config") -> dict:
         result = super().as_json_dict(config)
 
         if OutputType.samples in config.output_types:
-            result["samples"] = self.samples.tolist()
+            result["samples"] = Forecast.jsonify_floats(self.samples.tolist())
```

Comment:
I'm not sure what the right place for this call is. Maybe we want to add it to the `shell` module instead to ensure that it is always called on the result?

Reply:
My initial thought was that `as_json_dict` method - given the method name, should be outputting well formed JSON and hence added it there, but it makes more sense to do it right before sending the response.